src/themes/capybara/pages/Home.vue

```diff
@@ -0,0 +1,273 @@
+        :key="i"
+        :title="hero.title"
+        :subtitle="hero.subtitle"
+        :button-text="hero.buttonText"
+        :background="hero.background"
+        :image="hero.image"
+        :class="hero.className"
+      />
+    </SfHero>
+    <SfBannerGrid :banner-grid="1" class="banners">
+      <template #bannerA>
+        <SfBanner
+          subtitle="Dresses"
+          title="COCKTAIL PARTY"
+          description="Find stunning women's cocktail dresses and party dresses. Stand out in lace and metallic cocktail dresses from all your favorite brands."
+          button-text="SHOP NOW"
+          image="/assets/homepage/bannerF.png"
+          class="sf-banner--left sf-banner--container-full"
+        />
+      </template>
+      <template #bannerB>
+        <SfBanner
+          subtitle="Dresses"
+          title="LINEN DRESSES"
+          description="Find stunning women's cocktail dresses and party dresses. Stand out in lace and metallic cocktail dresses from all your favorite brands."
+          button-text="SHOP NOW"
+          image="/assets/homepage/bannerE.png"
+          class="sf-banner--left"
+        />
+      </template>
+      <template #bannerC>
+        <SfBanner
+          subtitle="T-Shirts"
+          title="THE OFFICE LIFE"
+          image="/assets/homepage/bannerC.png"
+          class="sf-banner--left sf-banner--container-full"
+        />
+      </template>
+      <template #bannerD>
+        <SfBanner
+          subtitle="Summer shoes"
+          title="ECO SANDALS"
+          image="/assets/homepage/bannerG.png"
+          class="sf-banner--left sf-banner--container-full"
+        />
+      </template>
+    </SfBannerGrid>
+    <SfCallToAction
+      title="Subscribe to Newsletters"
+      button-text="Subscribe"
+      description="Be aware of upcoming sales and events. Receive gifts and special offers!"
+      class="call-to-action-newsletter"
+      image="/assets/homepage/newsletter.png"
+    />
+    <SfSection
+      :heading="{
+        title: 'Best Sellers'
+      }"
+    >
+      <SfCarousel :settings="{ gap: 0 }" class="product-carousel">
+        <SfCarouselItem v-for="(product, i) in newProducts" :key="i">
+          <SfProductCard
+            :title="product.name"
+            :regular-price="product.priceInclTax | price"
+            class="product-card"
+          />
+        </SfCarouselItem>
+      </SfCarousel>
+    </SfSection>
+    <SfSection
+      :heading="{
+        title: 'Share Your Look',
+        subtitle: '#YOURLOOK'
+      }"
+    >
+      <div class="grid grid-images">
+        <div class="grid__row">
+          <div class="grid__col">
+            <SfImage src="/assets/homepage/imageA.png">
+              katherina_trn
+            </SfImage>
+          </div>
+          <div class="grid__col">
+            <SfImage src="/assets/homepage/imageB.png">
+              katherina_trn
+            </SfImage>
+          </div>
+        </div>
+        <div class="grid__row">
```

Comment:
is there any place where the classes have been pre-defined? @filrak to make it easier for developers to pick the right one

Reply:
what you mean by predefined?

Reply:
All classes are encapsulated in Home.vue